### ISSUE TITLE
Privacy Badger Chrome forgets settings after a few weeks - cannot enable tracking again

### ISSUE THREAD

**Issue by poppyuser** [EFForg/privacybadgerchrome#370]
_From @poppyuser on April 17, 2015 19:17_

I installed Privacy Badger and loved it for a few weeks. I blocked cookies or sites just fine.

But after a few weeks I noticed it stopped working. All tracking would show as green. Clicking on yellow or red would not work (would not be remembered).

I ended up having to remove the plugin and reinstall.
It worked again perfectly but only for a few weeks. Today again I see everything is green. it will not 'remember' the settings if I change anything to yellow or red.

_Copied from original issue: EFForg/privacybadgerfirefox#338_

**Comment by poppyuser** [comment]
_From @PiRSquared17 on April 17, 2015 19:20_

Hi, please move this bug report to https://github.com/EFForg/privacybadgerchrome unless it applies to Privacy Badger Firefox too. Thanks for reporting it.

**Comment by poppyuser** [comment]
The only two enabled extensions I have installed is PB and Cisco Webex Extension. I have others but all disabled. Running windows 8. Chrome Version 43.0.2357.18 beta-m (64-bit) - up to date.

**Comment by SwartzCr** [comment]
My guess is that privacy badger is having trouble storing its settings
It'd be great if you could look at the console for it and share any error messages you see.
To do this, click on the bader icon, right click in the dropdown window and select inspect element and then click the 'console' tab of it.
The other place you should look is the background.js scope. Go to chrome://extensions/ and click the "Inspect Views: background page, and select the 'console' tab there too.
If you see any errors in either of these please please please share them here.
This is likely a bug and it would be wonderful to be able to fix it.
In the short term to get PB working again I'd recommend removing and then re-installing the extension, but this will obviously re-set all of the settings it had.

**Comment by poppyuser** [comment]
I clicked on icon and selected inspect element. I then clicked console tab.
It shows '>' char. The dropdown above it has one option only <Top Frame>.
Nothing appearing.
I checked Preserve Logs and reloaded the site I was on (which had 6 trackers). 
Tried going to yellow on some of the trackers and to red on others. Then reloaded the page. Nothing still appearing in that console window.

**Comment by SwartzCr** [comment]
Odd! And it's still not saving your settings?
Can you try doing this, while in that console type:
FilterStorage.knownSubscriptions.userRed
FilterStorage.knownSubscriptions.userYellow
this will show you the sites that privacy badger is aware you've changed the settings for.
Then change the settings for some sites and see if they show up. Let me know what happens!

**Comment by mlane3** [comment]
I recently experience this same error.  I am not sure how reproducible it is.   I tried deleting cookies (and everything except my passwords), desyncing chrome after deleting cookies, disabling and re-enabling extensions.  The only thing that eventually seemed to work was uninstalling Privacy Badger, desyncing, and then hitting reset settings to default.  I think it might be fixed, but I am not sure.

However, its still acting a bit wonky as its not blocking everything I would expect it to block.  For example, going to whitehouse.gov/privacy "no trackers" show up or only one tracker like twitter will show itself up as not blocked. Also "Send DNT message" or "block-third party cookies", does not automatically turn on when privacy badger loads.

  Is their a test webpage I can go to just filled with trackers normally blocked by privacy badger, so I can see if its fixed?

P.S.  I am not sure if this is relevant or not.  But, the night before google was have some "Error: "He's Dead, Jim!"" on some google searches while I was in incognito mode (unlike poppyuser).  I normally have Script Safe, Ghostry, and Privacy Badger enabled during those searches.  Eventually Script Safe starting having cookie errors and was crashing chrome, so I had to disable it.

List of extensions I use that I have currently disabled:
Bookmark Manager 2.2015.611.10417
Capture Webpage Screenshot - FireShot 0.98.68
Chrome Speak 1.2.4.1
DEPRECATED: Keep My Opt-Outs 2.0.1
EditThisCookie 1.4.1
Ghostery 5.4.3
Google Docs 0.9
Google Scholar Button 1.4
Google Sheets 1.1
Google Slides 0.9
Personal Blocklist (by Google) 2.6
Print Friendly & PDF 2.3
Print or PDF with CleanPrint 4.8.4
Privacy Badger 2015.4.1
ScriptSafe 1.0.6.18
SpeakIt! 0.2.96 
TLDR 2.1.1  (This extension is broken q-q, so I have not be able to use it in forever)

**Comment by cooperq** [comment]
this is a duplicate of #353 so I am going to close it and address the issue over there.
